meetings/working-groups/collection-literals/meetings/working-groups/collection-literals/CL-2023-06-16.md

```diff
@@ -0,0 +1,125 @@
+# Collection literals working group meeting for June 26, 2023
+
+## Agenda
+
+* Collection expression lifetimes
+* BCL apis
+* Breaking changes
+* Extension method lookup
+* Syntactic disambiguation
+
+## Discussion
+
+### Collection expression lifetimes
+
+Relevant link: https://gist.github.com/jaredpar/a784f8963ce73613a0a696ef89a00d9a
+
+Discussed the rules about spans produced by collection literals, and whether those are 'safe to return' from the 'current method'.
+
+Conclusion.  We will accept the rules in the provided link. Specifically, spans from collection literals are *not* "Safe to return from current method". This has the benefit of allowing the compiler to potentially stackalloc these spans for very high efficiency code (including cases like `G([1, 2, 3])`).
```

Comment:
I think there was a second point that the very high-efficiency code is only possible when passing to a `scoped` method argument.

Reply:
feel free to suggest whatever change you'd like :)